mlshvch/hometask-2/get_middle_character.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+def get_middle(string)
+  middle_string_position = string.length / 2
+  string.length.even? ? string[(middle_string_position - 1)..middle_string_position] : string[middle_string_position]
+end
+
+EXIT_COMMAND = 'exit!'
```

Comment:
Usually, for the sake of convenience, the constants are defined on top of class / module definition, or, in cases with the plain script files, in the beginning of file itself. It's something of an unwritten convention.